<launch>
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
        <!-- 代价地图 -->
        <rosparam file="$(find agv_nav)/param/move_base/costmap/costmap_common_params.yaml" 
                  command="load" 
                  ns="global_costmap" />
        <rosparam file="$(find agv_nav)/param/move_base/costmap/costmap_common_params.yaml" 
                  command="load" 
                  ns="local_costmap" />
        <rosparam file="$(find agv_nav)/param/move_base/costmap/local_costmap_params.yaml" 
                  command="load" />
        <rosparam file="$(find agv_nav)/param/move_base/costmap/global_costmap_params.yaml" 
                  command="load" />
        <!-- 规划器 -->
        <rosparam file="$(find agv_nav)/param/move_base/move_base_params.yaml" 
                  command="load" />
        <rosparam file="$(find agv_nav)/param/move_base/planner/dwa_local_planner_params.yaml" 
                  command="load" />
        <rosparam file="$(find agv_nav)/param/move_base/planner/global_planner_params.yaml" 
                  command="load" />
    </node>
</launch>